src/vs/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -441,6 +456,30 @@ export class NotebookEditor extends EditorPane implements INotebookEditorPane, I
 			}
 		}
 
+		// Notebook information
+		let codeCellCount: number | undefined = undefined;
+		let mdCellCount: number | undefined = undefined;
+		let outputCount: number | undefined = undefined;
+		let outputBytes: number | undefined = undefined;
+		let codeLength: number | undefined = undefined;
+		let markdownLength: number | undefined = undefined;
+		let notebookStatsLoaded: number | undefined = undefined;
+		if (notebook) {
+			const stopWatch = new StopWatch();
+			for (const cell of notebook.cells) {
+				if (cell.cellKind === CellKind.Code) {
+					codeCellCount = (codeCellCount || 0) + 1;
+					codeLength = (codeLength || 0) + cell.getTextLength();
+					outputCount = (outputCount || 0) + cell.outputs.length;
+					outputBytes = (outputBytes || 0) + cell.outputs.reduce((prev, cur) => prev + cur.outputs.reduce((size, item) => size + item.data.byteLength, 0), 0);
+				} else {
+					mdCellCount = (mdCellCount || 0) + 1;
+					markdownLength = (codeLength || 0) + cell.getTextLength();
+				}
+			}
+			notebookStatsLoaded = stopWatch.elapsed();
```

Comment:
Added this to see how much of an impact this will have, 
If we find out that this is slow, we can always disable this.
Also from my testing in OSS, this takes less than 1ms.